config/v1/types_cluster_version.go

```diff
@@ -95,6 +95,13 @@ type ClusterVersionStatus struct {
 	// +required
 	Desired Release `json:"desired"`
 
+	// architecture indicates whether the cluster is a homogeneous or
```

Comment:
Once we support multi, why would a user ever want to choose single?

Reply:
This will also be set upon initial install/CVO startup which could be occurring with an homogeneous payload.